samples/grid-pro/cypress/custom-sorting/demo.js

```diff
@@ -0,0 +1,43 @@
+const sorter = (a, b) => {
+    const isA40 = a === 40 || a === '40';
+    const isB40 = b === 40 || b === '40';
+
+    if (isA40 && !isB40) {
+        return -1;
+    }
+    if (isB40 && !isA40) {
+        return 1;
+    }
+    if (isA40 && isB40) {
+        return 0;
+    }
+
+    const isNumA = typeof a === 'number';
+    const isNumB = typeof b === 'number';
+
+    if (isNumA && isNumB) {
+        return a - b;
+    }
+
+    return isNumA ? -1 : 1;
+};
+
+DataGrid.dataGrid('container', {
```

Comment:
Use `Grid.grid` instead.

FYI: Currently, the DataGrid is only fallback to Grid.